Formula/gsoap.rb

```diff
@@ -15,12 +15,6 @@ class Gsoap < Formula
   def install
     ENV.deparallelize
```

Comment:
Disappointing but thanks for checking. Could you add a Ruby inline comment just noting when you contacted them, how and what they said (abbreviated version; a line or two max). Thanks!